bin/sparql-benchmark-runner.ts

```diff
@@ -61,6 +61,11 @@ async function main(): Promise<void> {
         coerce: (arg: number) => arg * 1_000,
         number: true,
       },
+      outputRawResults: {
+        type: 'boolean',
+        description: 'A flag indicating if SPARQL Benchmark Runner should also output the raw results',
```

Comment:
```suggestion
        description: 'A flag indicating if raw results should be produced',
```

But perhaps "raw results" is too vague (also as flag name).
I'm still not sure what these raw results are even.
Could you also include a brief example of this in the README?